ping/ping_common.c

```diff
@@ -521,7 +521,7 @@ void setup(struct ping_rts *rts, socket_st *sock)
 	}
 
 	if (sock->socktype == SOCK_RAW)
-		rts->ident = htons(getpid() & 0xFFFF);
+		rts->ident = rand() & 0xFFFF;
```

Comment:
It'd be worth to mention in commit message, that it's only for `SOCK_RAW`. For `SOCK_DGRAM` is id sequential (probably handled by protocol, i.e. kernel icmp implementation).

"historycally" => historically :)

I guess `rand()` is ok for us

Reply:
You mean the one enabled by `net.ipv4.ping_group_range` right?
unfortunately all distro disables it due to past security issues